client/app/app.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+angular.module('throwcast', [
+  'throwcast.playlist',
+  'throwcast.categories',
+  'throwcast.popular',
+  'throwcast.auth',
+  'ngRoute'
```

Comment:
I actually think this will fail since the angular-routes dependency isn't installed.